docs/example.md

```diff
@@ -51,47 +54,45 @@ This has been tested on ADB 1.7.1 as Fedora 23 on local machine
 
 ## Use Openshift2Nulecule to export sample application
  1. Install Openshift2Nulecule
-    - Add ADB testing repo (required only for ADB 1.7.0)
-        ```
-        sudo sh -c "cat > /etc/yum.repos.d/centos7-adb-cbs-test.repo <<EOF
-        [centos7-adb-cbs-test]
-        name=CentOS-$releasever - ADB CBS Testing
-        baseurl=https://cbs.centos.org/repos/atomic7-adb-common-testing/x86_64/os/
-        gpgcheck=0
-        enabled=1
-        EOF
-        "
-        ```
+
+    This step is **not** required if you are using CDK 2.0. OpenShift2Nulecule should be already installed in CDK box.
+
+    - Add [tkral/openshift2nulecle](https://copr.fedorainfracloud.org/coprs/tkral/openshift2nulecle/) Copr repository
+      
+      `sudo sh -c "curl  https://copr.fedorainfracloud.org/coprs/tkral/openshift2nulecle/repo/epel-7/tkral-openshift2nulecle-epel-7.repo > /etc/yum.repos.d/tkral-openshift2nulecle-epel-7.repo"`
```

Comment:
I feel we should add the install information in https://github.com/projectatomic/openshift2nulecule/blob/master/docs/installation.md#usage-installation 

Here you can divide the 'Usage installation' into two as, "Install as RPM" and "Install from Source". So the above steps add under "Install as RPM" and here just put a link to that topic.